mne/report/report.py

```diff
@@ -3934,9 +3946,16 @@ def _add_epochs(
         ch_types = _get_data_ch_types(epochs)
         epochs.load_data()
 
+        if image_kwargs is None:
+            image_kwargs = dict(zip(ch_types, [None]*len(ch_types)))
+
         for ch_type in ch_types:
             with use_log_level(_verbose_safe_false(level="error")):
-                figs = epochs.copy().pick(ch_type, verbose=False).plot_image(show=False)
+                figs = (
+                    epochs.copy()
+                    .pick(ch_type, verbose=False)
+                    .plot_image(show=False, **image_kwargs[ch_type]) # SH: not sure if this works
```

Comment:
Testing should tell us! But really let's not require each ch_type to be present...
```suggestion
                    .plot_image(show=False, **image_kwargs.pop(ch_type, dict()))
```

Reply:
And then outside the loop make sure `image_kwargs` is empty, otherwise the user has done something bad (e.g., passed `image_kwargs=dict(colorbar=False)` when they needed to do `image_kwargs=dict(mag=dict(colorbar=True))`